src/App.tsx

```diff
@@ -1,24 +1,228 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Notification } from './components/Notification';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { ToodList } from './components/TodoList/TodoList';
+import {
+  getTodos,
+  postTodo,
+  deleteTodo, patchTodo,
+} from './api/todos';
+import { Loader } from './components/Loader';
+import { FilterType } from './types/FilterEnum';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+const USER_ID = 7006;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [title, setTitle] = useState('');
```

Comment:
why do you need this state in this component? move form related state to component with state and just pass function that should be run when form is submited